theme-data/dark/dark-common.json

```diff
@@ -203,6 +203,14 @@
     "overlay": {
       "meetings": {
         "normal": "@color-black-alpha-70"
+        },
+      "audiodump": {
+        "primary": {
+          "normal": "@color-white-alpha-11"
+        },
```

Comment:
most of the things needed by audiodump are already defined and i don't think needs to be added. Components should be general things that are used to build a feature.